src/Renci.SshNet.Tests/Classes/ShellStreamTest.cs

```diff
@@ -103,6 +103,36 @@ public void WriteLine_Line_ShouldOnlyWriteLineTerminatorWhenLineIsNull()
             _channelSessionMock.Verify(p => p.SendData(lineTerminator), Times.Once);
         }
 
+        [TestMethod]
+        public void WindowChangeRequest_ShouldReturnFalseWhenNotConnected()
+        {
+            var shellStream = CreateShellStream();
+            _channelSessionMock.Setup(s => s.IsOpen).Returns(false);
+            Assert.IsFalse(shellStream.WindowChangeRequest(80, 25, 0, 0));
+        }
+
+        [TestMethod]
+        public void WindowChangeRequest_ShouldReturnFalseWhenResultFails()
+        {
+            var shellStream = CreateShellStream();
+            _channelSessionMock.Setup(s => s.IsOpen).Returns(true);
+            _channelSessionMock.Setup(s => s.SendWindowChangeRequest(
```

Comment:
Not very useful since `SendWindowChangeRequest(...)` always returns **true**, but let's discuss.

Reply:
Hmm, I dont think SendWindowChangeRequest SHOULD always be true.  For example a change to size of 0,0,0,0 should fail and probably will on some pty implementations.